app/app/src/main/res/layout/content_joystick.xml

```diff
@@ -15,35 +39,39 @@
         android:visibility="invisible" />
 
     <!-- Forward button -->
+
+    <!-- Backward button -->
+
+    <!-- Left button -->
+
+    <!-- Right button -->
+
     <ImageButton
         android:id="@+id/forwardBtn"
         style="@style/controlBtn"
         android:layout_above="@id/whitespace"
-        android:layout_centerHorizontal="true"
+        android:layout_marginBottom="34dp"
         android:src="@drawable/nav_forward" />
 
-    <!-- Backward button -->
     <ImageButton
```

Comment:
This button should not be included in the layout if there is a joystick to replace it.